l10n_ch_payment_slip/views/account_invoice.xml

```diff
@@ -1,71 +1,69 @@
-            class="oe_highlight"/>
-        </xpath>
-        <xpath expr="//button[@name='invoice_print'][2]" position="after">
-          <button type="object"
-            string="BVR/ESR"
-            name="print_bvr"
-            attrs="{'invisible':['|',('sent','=',False), ('state', '!=', 'open')]}"/>
-        </xpath>
-        <field name="tax_line_ids" position="before">
-          <group colspan="4">
-            <field name="bvr_reference" colspan="4"/>
-            <button type="action"
-              string="Open Payment Slips"
-              groups="base.group_system"
-              name="%(l10n_ch_payment_slip.action_slip_all)d"
-              attrs="{'invisible': ['|', ('type', '!=', 'out_invoice'), ('bvr_reference', '=', False)]}"/>
-          </group>
+    <record model="ir.ui.view" id="invoice_form_add_bvr_ref">
+        <field name="name">account.invoice.form.bvr.ref</field>
+        <field name="model">account.invoice</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="account.invoice_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='invoice_print'][1]" position="after">
+                <button type="object"
+                    string="BVR/ESR"
+                    name="print_bvr"
+                    attrs="{'invisible': ['|',('sent','=',True), ('state', '!=', 'open')]}"
+                    class="oe_highlight"/>
+            </xpath>
+            <xpath expr="//button[@name='invoice_print'][2]" position="after">
+                <button type="object"
+                    string="BVR/ESR"
+                    name="print_bvr"
+                    attrs="{'invisible':['|',('sent','=',False), ('state', '!=', 'open')]}"/>
+            </xpath>
+            <field name="tax_line_ids" position="before">
+                <group colspan="4">
+                    <field name="bvr_reference" colspan="4"/>
+                    <button type="action"
+                        string="Open Payment Slips"
+                        groups="base.group_system"
+                        name="%(l10n_ch_payment_slip.action_slip_all)d"
+                        attrs="{'invisible': ['|', ('type', '!=', 'out_invoice'), ('bvr_reference', '=', False)]}"/>
+                </group>
+            </field>
         </field>
-      </field>
     </record>
 
     <record id="pay_slip_form" model="ir.ui.view">
-      <field name="name">pay slip form</field>
-      <field name="model">l10n_ch.payment_slip</field>
-      <field name="arch" type="xml">
-        <form string="Payment Slip">
-          <group>
-            <field name="reference"/>
-            <field name="move_line_id"/>
-            <field name="amount_total"/>
-            <field name="scan_line"/>
-            <field name="slip_image"/>
-            <field name="a4_pdf"/>
-          </group>
-        </form>
-      </field>
+        <field name="name">pay slip form</field>
```

Comment:
Not required, but since we had `openerp` and `data` element in v10, it make sense replacing with odoo. As it's one element instead of two, some reformatting make sense IMO.

Reply:
Then, what are we supposed to do when changing  `<openerp> <data>` to `<odoo>` ? 
Leave 4 spaces after first element (odoo) and 2 afterwards, or reindent with 2 spaces ?

Reply:
Indeed, you are right it would break for cherry-pick anyway.

Still, please keep the 2 spaces. It is still readable and save space in view definition that can have several level of xml. And when opening a file and seeing only spaces it doesn't help.

Or please explain why you want to change it.